apps/codeforafrica/src/lib/api.netlify-cms/getTeam.js

```diff
@@ -0,0 +1,15 @@
+import { join } from "path";
+
+import getCollectionData from "./getCollectionData";
+
+const teamDir = join(process.cwd(), "content/team");
+
+export default function getTeam(fields) {
+  const teams = getCollectionData(teamDir, fields);
+  teams.map((team) => {
+    const member = team;
+    member.href = `/about/members/${member.slug}`;
+    return member;
+  }, []);
+  return teams;
```

Comment:
```suggestion
  return teams.map((member) => {
    member.href = `/about/members/${member.slug}`;
    return member;
  });
```

Reply:
The reason I didn't do this @kilemensi  was because of eslint error
` Assignment to property of function parameter `
should I just ignore it?

Reply:
This is the correct way @esirK ... To please eslint,
```js
  return teams.map((member) => {
    return { ...member, href: `/about/members/${member.slug}` };
  });
```